### PR TITLE
Update of the French translation

### PR DESCRIPTION
**Pull request by Yamakuni**
Add and modify some little things

### CODE REVIEW

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 1]
```diff
@@ -177,7 +178,7 @@ msgstr "Mise à jour terminée, merci d’appuyer sur okay et de rafraîchir la
 
 #: cps/admin.py:685 cps/admin.py:686 cps/admin.py:687 cps/admin.py:688
 msgid "Update failed:"
-msgstr "La mise à jour à échouée : "
+msgstr "La mise à jour à échouée :"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "La mise à jour a échoué :"
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 2]
```diff
@@ -229,15 +230,15 @@ msgstr "Le format de fichier %(ext)s a été ajouté à %(book)s"
 
 #: cps/editbooks.py:373
 msgid "Cover is not a supported imageformat (jpg/png/webp), can't save"
-msgstr ""
+msgstr "La couverture n’est pas un format d’image supporté (jpg/png/webp), impossible d’enregistrer"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Le format d'image utilisé pour la couverture n'est pas supporté (jpg/png/webp uniquement). Sauvegarde impossible."
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -283,7 +284,7 @@ msgstr "Une erreur est survenue au cours de la conversion du livre : %(res)s"
 
 #: cps/gdrive.py:61
 msgid "Google Drive setup not completed, try to deactivate and activate Google Drive again"
-msgstr ""
+msgstr "La configuration du lecteur Google n’est pas terminée, essayez de désactiver et d’activer à nouveau le lecteur Google"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "La configuration de Google Drive n’est pas terminée, essayez de désactiver et d’activer à nouveau Google Drive."
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 4]
```diff
@@ -708,7 +709,7 @@ msgstr "Livres au hasard"
 
 #: cps/web.py:522
 msgid "Books"
-msgstr ""
+msgstr "LIvres"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Livres"
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 5]
```diff
@@ -687,7 +688,7 @@ msgstr "Une nouvelle mise à jour est disponible. Cliquez sur le bouton ci-desso
 
 #: cps/updater.py:423
 msgid "Click on the button below to update to the latest stable version."
-msgstr ""
+msgstr "Cliquez sur le bouton ci-dessous pour mettre à jour la dernière version stable."
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Téléchargez la dernière version en cliquant sur le bouton ci-dessous."
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 6]
```diff
@@ -736,12 +737,12 @@ msgstr "Séries : %(serie)s"
 #: cps/web.py:607
 #, python-format
 msgid "Rating: %(rating)s stars"
-msgstr ""
+msgstr "Notes: %(rating)s étoiles"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Note: %(rating)s étoiles"
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 7]
```diff
@@ -1616,11 +1617,11 @@ msgstr "langue"
 
 #: cps/templates/detail.html:200
 msgid "Mark As Unread"
-msgstr ""
+msgstr "Marquer comme non-lu"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Marquer comme non lu"
```

File: cps/translations/fr/LC_MESSAGES/messages.po [hunk 8]
```diff
@@ -1705,7 +1706,7 @@ msgstr "Chercher"
 
 #: cps/templates/http_error.html:37
 msgid "Create issue"
-msgstr ""
+msgstr "Faire un signalement"
```
Comment (Zenigata) [inline cps/translations/fr/LC_MESSAGES/messages.po]:
```suggestion
msgstr "Signaler un problème"
```

**Comment by Yamakuni** [comment]
> As soon as this review is merged, I will add another one.

Thank you for your corrections.

**Pull request merged**